contents.json

```diff
@@ -16917,17 +16917,22 @@
         "github",
         "rxswift"
       ],
-      "description": "GitHub app using RxSwift and MVVM",
+      "description": "GitHub iOS client written in RxSwift and MVVM clean architecture",
```

Comment:
Hi @khoren93, could you remove "iOS" from description ([Avoid iOS or open-source in description as it is assumed](https://github.com/dkhamsing/open-source-ios-apps/blob/master/.github/PULL_REQUEST_TEMPLATE.md))?